app/templates/discover.hbs

```diff
@@ -26,7 +26,7 @@
         </div>
     </div>
     {{#if queryBody}}
-        <div class="row" style="text-align: center; font-size:75%; margin-top:15px">
+        <div class="row hidden-xs" style="text-align: center; font-size:75%; margin-top:15px">
```

Comment:
They couldn't see both, but what if (on `xs` screens) the facets were hidden by default, with a "Filter" button/link that toggles them? When displayed, they'd be stacked above the search results, which could be annoying to users but they'd have to ask for it.